styles.css

```diff
@@ -0,0 +1,89 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+@media only screen and (max-width: 800px) {
+  body {
+    background-color: lightblue;
+  }
+}
+
+nav {
+  position: fixed;
+  top: 0;
+  width: 100vw;
+  height: 10vh;
+  background-color: #be3144;
+}
+
+nav ul {
+  display: flex;
+  justify-content: flex-end;
+}
```

Comment:
Your navigation tabs are correctly horizontally positioned, but it would look better if you also positioned them vertically in the flexbox.
That will help to make your navigation menu look good on all devices